sdks/python/run_postcommit.sh

```diff
@@ -70,22 +70,6 @@ SDK_LOCATION=$(find dist/apache-beam-*.tar.gz)
 echo "pyhamcrest" > postcommit_requirements.txt
 echo "mock" >> postcommit_requirements.txt
```

Comment:
Do we still use `postcommit_requirements.txt` in this file?

Reply:
thanks for the catch. I checked existing validatesrunner tests and none of them use `pyhamcrest` and `mock`. Will remove it.